Core/Compatibility.cpp

```diff
@@ -83,5 +93,7 @@ void Compatibility::CheckSettings(IniFile &iniFile, const std::string &gameID) {
 }
 
 void Compatibility::CheckSetting(IniFile &iniFile, const std::string &gameID, const char *option, bool *flag) {
-	iniFile.Get(option, gameID.c_str(), flag, *flag);
+	if (ignored_.find(option) != ignored_.end()) {
```

Comment:
This is backwards, heh. Should ignore it if it finds it, not apply it if it find it :)

Reply:
Also this is why sane vector libraries has ".contains()" methods, so much more readable than this find iterator stuff :)

Reply:
Bah, sorry.

-[Unknown]